<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataWindow</class>
 <widget class="QMainWindow" name="DataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1091</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{
	border-image: url(:/IMG_2027.JPG);}</string>
    </property>
    <widget class="QLineEdit" name="temperatureLineEdit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>230</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 150, 150,120);
	color: rgb(255, 255, 255);

border:1px solid #e1e0e0;
	border-color: rgb(0, 170, 255);
	font: 18pt &quot;黑体&quot;;
	border-radius:3px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>230</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 0);
font: 12pt &quot;黑体&quot;;
background-color: rgb(150, 150, 150,120);
	color: rgb(255, 255, 255);
color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bloodGluseLineEdit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>280</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(141, 141, 141,120);
	color: rgb(255, 255, 255);

border:1px solid #e1e0e0;
	border-color: rgb(0, 170, 255);

	font: 18pt &quot;黑体&quot;;
	border-radius:3px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>280</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
font: 12pt &quot;黑体&quot;;
background-color: rgb(150, 150, 150,120);
</string>
     </property>
     <property name="text">
      <string>血糖</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bloodOxygenLineEdit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>340</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(141, 141, 141,120);
	color: rgb(255, 255, 255);

border:1px solid #e1e0e0;
	border-color: rgb(0, 170, 255);

	font: 18pt &quot;黑体&quot;;
	border-radius:3px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>340</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
font: 12pt &quot;黑体&quot;;
background-color: rgb(150, 150, 150,120);
</string>
     </property>
     <property name="text">
      <string>血氧</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="heartRateLineEdit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>400</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(141, 141, 141,120);
	color: rgb(255, 255, 255);

border:1px solid #e1e0e0;
	border-color: rgb(0, 170, 255);

	font: 18pt &quot;黑体&quot;;
	border-radius:3px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>400</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
font: 12pt &quot;黑体&quot;;
background-color: rgb(150, 150, 150,120);
</string>
     </property>
     <property name="text">
      <string>心率</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bloodPresureLineEdit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>450</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(141, 141, 141,120);
	color: rgb(255, 255, 255);

border:1px solid #e1e0e0;
	border-color: rgb(0, 170, 255);

	font: 18pt &quot;黑体&quot;;
	border-radius:3px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>450</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
font: 12pt &quot;黑体&quot;;
background-color: rgb(150, 150, 150,120);
</string>
     </property>
     <property name="text">
      <string>血压</string>
     </property>
    </widget>
    <widget class="QPushButton" name="commitPushButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>540</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 170, 127);</string>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;Arial&quot;;
color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>请输入体征数据</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
